### ISSUE TITLE
TypeError: Cannot read property 'prefix_exceptions' of undefined

### ISSUE THREAD

**Issue by miguelmota** [browserslist/caniuse-lite#72]
Hi, I have a create-react-app application and I started seeing this build error recently coming from the caniuse-lite node module.

```bash
11:18:33 PM: Creating an optimized production build...
11:23:37 PM: Failed to compile.
11:23:37 PM: 
11:23:37 PM: ./src/index.css
11:23:37 PM: TypeError: Cannot read property 'prefix_exceptions' of undefined
11:23:37 PM:     at Array.map (<anonymous>)
11:23:38 PM: npm ERR! code ELIFECYCLE
11:23:38 PM: npm ERR! errno 1
```

I downgraded to `caniuse-lite@1.0.30001223` and build works but the latest version is not working.

**Comment by xqin** [comment]
https://github.com/browserslist/caniuse-lite/issues/70

**Comment by ai** [comment]
Should be fixed in  1.0.30001226

**Comment by miguelmota** [comment]
Thanks,and  sorry for duplicate issue.
